extensions/pyRevitTools.extension/checks/radar_check.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+
+import math
+# pyRevit
+from pyrevit import revit, DB, DOCS, HOST_APP
+from pyrevit import script
+from pyrevit.preflight import PreflightTestCase
+
+
+doc = DOCS.doc  # Current Document
+INTERNAL_ORIGIN = (0, 0, 0)
+EXTENT_DISTANCE = 52800  # Linear Feet
+BAD_STRG = '### :thumbs_down_medium_skin_tone: ............'
+GOOD_STRG = '### :OK_hand_medium_skin_tone: ............'
+WARN_STRG = '### :warning: ............'
+CRITERIA_STRG = '10 Mi (16KM) away from the Internal Origin.'
+
+# ___________________________________________________3D to Bounding Box Analysis
+
+def get_tempbbox(toggle_cads, toggle_rvts, toggle_ifcs, toggle_all):
+    bad_cads = []
+    bad_rvts = []
+    bad_elements = []
+    # Create a 3D view
+    with revit.DryTransaction("Create 3D View"):
+        view_3D_types = (DB.FilteredElementCollector(doc)
+                        .OfClass(DB.ViewFamilyType)
+                        .WhereElementIsElementType()
+                        .ToElements())
```

Comment:
This is an interesting replacement for the FilteredElementCollector, do you know how to get the output to be a List of ElementIds instead of an element?

Reply:
Unfortunately, at the moment, the pyrevit functions only return the elements.
If you only need the elementids, just keep the `FilteredElementCollector` and use `ToElementIds()` method.